publish_python/upload/packagecloud.py

```diff
@@ -24,7 +24,7 @@ def upload_packagecloud(*, artifacts, upload, config):
         destination = f'{repository}{version}'
         for artifact in artifacts:
             ext = os.path.splitext(artifact)[1]
-            if ext not in ('.deb', '.whl'):
+            if ext not in ('.deb', '.whl', '.dsc'):
```

Comment:
Nitpick: can you please insert the new value in alphabetical order?

Reply:
sure, 4add04c